page89.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+age = 15
+
+if age < 2:
+    print("You're a baby.")
+
+elif age == 2 or age < 4:
```

Comment:
So this will get any age equal to 2 or _anything_ less than 4 (such as -1). A better approach is:

```
elif age >= 2 and age < 4:
```

This will apply to ages 2 and 3.